<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtGuiApplication1Class</class>
 <widget class="QMainWindow" name="QtGuiApplication1Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>953</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtGuiApplication1</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="rbHome">
        <property name="text">
         <string>Homeport</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbAlbum">
        <property name="text">
         <string>Album</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="PreviewWidget" name="preview" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>953</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Room">
    <property name="title">
     <string>Modify &amp;Room</string>
    </property>
    <addaction name="actionChange_Wall"/>
    <addaction name="actionChange_Floor"/>
    <addaction name="actionChange_Desk"/>
    <addaction name="actionChange_Window"/>
    <addaction name="actionChange_Object"/>
    <addaction name="actionChange_Chest"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionSavePreview"/>
    <addaction name="actionExportSwf"/>
   </widget>
   <addaction name="menu_Room"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="shipDock">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Ships</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="ShipDockWidget" name="shipDockWidget" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTabWidget" name="tabDetails"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionChange_Wall">
   <property name="text">
    <string>Change W&amp;all</string>
   </property>
  </action>
  <action name="actionChange_Floor">
   <property name="text">
    <string>Change &amp;Floor</string>
   </property>
  </action>
  <action name="actionChange_Desk">
   <property name="text">
    <string>Change &amp;Desk</string>
   </property>
  </action>
  <action name="actionChange_Window">
   <property name="text">
    <string>Change &amp;Window</string>
   </property>
  </action>
  <action name="actionChange_Object">
   <property name="text">
    <string>Change &amp;Object</string>
   </property>
  </action>
  <action name="actionChange_Chest">
   <property name="text">
    <string>Change &amp;Chest</string>
   </property>
  </action>
  <action name="actionExportSwf">
   <property name="text">
    <string>&amp;Export Image From Swf </string>
   </property>
  </action>
  <action name="actionSavePreview">
   <property name="text">
    <string>&amp;Save Preview</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PreviewWidget</class>
   <extends>QWidget</extends>
   <header>PreviewWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ShipDockWidget</class>
   <extends>QWidget</extends>
   <header>ShipDockWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="KanColl.qrc"/>
 </resources>
 <connections/>
</ui>
